tests/repository/test_find_directory.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
```

Comment:
The current convention is to name the test filename after the function it tests, so this should be more like `test_determine_repo_dir_finds_subdirectories`?